projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -1051,7 +1051,7 @@ dataJob:
         # Case sensitive CSV list of write to's sources. Default is "DB,K8S" The control service
         # will write data job properties to all sources present in this list.
         # See the synchronizationTask for more information.
-        writeTos: "DB,K8S"
+        writeTos: "K8S"
```

Comment:
I'm just back from a long break off work. 

I think the the documentation here is pretty good but it would be soo much more useful if you guys said when you would you the different configurations. 

Am I correct in saying that with a fresh deployment with no exisitng jobs it would be best to use 

writeTos: "DB,K8S"
readDataSource: "DB"


If you are upgrading a control service for an existing deployment how should you set them?

Reply:
You are correct, but the new deployment functionality is still experimental, and we don't want to break existing deployments. We will introduce a tool for populating the missing data for the existing deployments.